### ISSUE TITLE
Error while starting media-dupes-0.10.2-win

### ISSUE THREAD

**Issue by awrog** [yafp/media-dupes#139]
```
TypeError: $(...).modal is not a function
    at HTMLDocument.<anonymous> (file:///C:/Program%20Files/media-dupes/resources/app/app/index.html:290:25)
    at mightThrow (C:\Program Files\media-dupes\resources\app\node_modules\jquery\dist\jquery.js:3762:29)
    at process (C:\Program Files\media-dupes\resources\app\node_modules\jquery\dist\jquery.js:3830:12)
    at sentryWrapped (C:\Program Files\media-dupes\resources\app\node_modules\@sentry\browser\dist\helpers.js:74:23)
```

---

media-dupes 0.10.2
Electron 9.1.2
win32 10.0.18363
Locale: en-US

**Comment by awrog** [comment]
Reverting to media-dupes-0.10.1-win
The program starts normally but still throws an error when I try to update de YouTube downloader

**Comment by yafp** [comment]
Hi & thanks for reporting this bug.

i just realized the error as well - did a minor mistake in the dependency list with bootstrap.

**Comment by yafp** [comment]
see #140

**Comment by yafp** [comment]
Regarding the new error: should be fixed with 0.10.3

Regarding the certificate error when trying to update youtube-dl - i am still waiting for feedback from the related developer.
